### PR TITLE
[fix] Helpful "Learn More" answer for "Do you want to add an admin queries API?" 

### PR DESCRIPTION
**Pull request by sw-yx**
*Description of changes:*

This prompt is duplicated from the prompt above it. It is very frustrating to hit "Learn More" and get an unhelpful answer, it is beyond frustrating to get a duplicate answer.

### CODE REVIEW

File: packages/amplify-category-auth/src/provider-utils/supported-services.ts [hunk 1]
```diff
@@ -308,7 +308,7 @@ export const supportedServices = {
       {
         key: 'adminQueries',
         question: 'Do you want to add an admin queries API?',
-        learnMore: 'This flow will help you add multiple user pool groups to your user-pool',
```
Comment (sw-yx) [inline packages/amplify-category-auth/src/provider-utils/supported-services.ts:311]: this is duplicated from line 296

**Comment by sw-yx** [comment]
i'm guessing @renebrandel will be able to fix the copy on this one

File: packages/amplify-category-auth/src/provider-utils/supported-services.ts [hunk 2]
```diff
@@ -308,7 +308,7 @@ export const supportedServices = {
...
+        learnMore: 'TODO: Find out what admin queries APIs are',
```
Comment (sw-yx) [inline packages/amplify-category-auth/src/provider-utils/supported-services.ts:311]:
```suggestion
        learnMore: 'Admin Queries API let you perform user admin functions from your frontend. See https://docs.amplify.aws/cli/auth/admin#admin-queries-api for more.',
```

**Review by renebrandel — APPROVED** [review]

**Review by kaustavghosh06 — APPROVED** [review]
LGTM.

**Pull request merged**